docs/app/files/Tags.mdx

```diff
@@ -54,7 +54,7 @@ To tag a file within a flow, use the `add_file_tag` function in the `ganymede_sd
 - _param_ **input_file_path**: str - The name of the file to tag.  The file path can be obtained from the keys in the dictionary returned by the [retrieve_files](../../sdk/GanymedeClass#method-retrieve_files) for Ganymede class or by calling the [get_gcs_uri method](../../sdk/GanymedeClass#method-get_gcs_uri).
 - _param_ **tag_type_id**: str - The name of the tag type to apply to the file.
 - _param_ **display_value**: str - Value of the tag to display in the file browser.
-- _param_ **tag_id**: Optional[str] - Optional tag ID that can be used to reference tags in code.
+- _param_ **tag_id**: Optional[str] - The tag id that acts as a unique identifier for the tag. If not specified, a hash to uniquely identify the tag will be generated as the tag ID.
```

Comment:
Is this true? @carusooo I think tag id is just left blank in the db. I don't think there is ever a generated tag id
<img width="604" alt="image" src="https://github.com/user-attachments/assets/ab02ae53-c153-4f08-a232-86afc6b05331">

Reply:
@zachaugenfeld - I think you were seeing that default is null - so @joshasharpe I think you're right about this and that the documentation is incorrect; @zachaugenfeld - can you confirm?